content/docs/en/elements/components/tab-view.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: TabView
 apiRef: https://docs.nativescript.org/api-reference/classes/_ui_tab_view_.tabview
-contributors: [MisterBrownRSA, rigor789, eddyverbruggen]
+contributors: [MisterBrownRSA, rigor789, eddyverbruggen, ikoevska]
 ---
 
-The TabView component allows you to display content grouped into tabs.
+`<TabView>` is a navigation component that shows content grouped into tabs and lets users switch between tabs.
 
 ---
 
-```html
+```xml
```

Comment:
And lastly, we can keep this `html` since it's a Vue template, which is basically html.

Reply:
No complaints here. Will fix it.